README.md

```diff
@@ -166,6 +170,11 @@ the cluster id, and keypair:
 --r53-zone my.hosted.domain --rhsm-user my_redhat_user --rhsm-pass my_redhat_pass
 ```
 
+More complex version
+```
+./run.py --no-confirm --cluster-id openshift --num-nodes 1 --keypair default --r53-zone alberttwong.com --rhsm-user alwong@redhat.com --rhsm-pass n0n34u --region us-west-1 --ami ami-d1315fb1 --run-smoke-tests --num-smoke-test-users 5 
+```
```

Comment:
Why is this necessary? You are effectively duplicating the --help option with an example.